test/parallel/test-http-response-multi-content-length.js

```diff
@@ -24,7 +24,7 @@ function test(server) {
 {
   const server = http.createServer((req, res) => {
     res.setHeader('content-length', [2, 1]);
-    res.end('ok');
+    res.end('k');
```

Comment:
Why this change?

Reply:
The `.end()` throws if you try to write more data than the given `content-length`, in this case, the test was failing with the content length being 1 and we were writing 2 bytes to it.